core/src/session.cpp

```diff
@@ -88,7 +108,8 @@ namespace irods::http
 		// Process client request and send a response.
 		//
 
-		auto req_ = parser_->release();
+        // TODO the parser probably is invalidated?
```

Comment:
Let's remove TODO and capture in a comment if there's confusion around it.

Reply:
I have no idea why this was added.  I just removed it.